back/doc/dependency_decisions.yml

```diff
@@ -1,174 +1,180 @@
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2020-11-16 23:52:39.960950292 Z
 - - :permit
   - Simplified BSD
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2020-11-16 23:54:14.723953082 Z
 - - :permit
   - BSD 3-Clause
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2020-11-16 23:55:19.036737894 Z
 - - :license
   - activerecord-postgis-adapter
   - BSD 3-Clause
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2020-11-17 00:01:48.552672458 Z
 - - :license
   - rgeo-geojson
   - BSD 3-Clause
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2020-11-17 00:02:58.643770167 Z
 - - :permit
   - 2-clause BSDL
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2020-11-17 00:05:01.110510128 Z
 - - :license
   - rgeo
   - BSD 3-Clause
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2020-11-17 00:06:31.056107479 Z
 - - :license
   - rgeo-activerecord
   - BSD 3-Clause
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2020-11-17 00:06:56.743802694 Z
 - - :license
   - simplecov-rcov
   - MIT
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2020-11-17 00:08:53.943260529 Z
 - - :permit
   - CitizenLab Commercial License V1
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2021-02-24 13:52:04.031811239 Z
 - - :approve
   - admin_api
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2021-02-24 13:55:48.746984657 Z
 - - :approve
   - email_campaigns
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2021-02-24 13:56:17.996729810 Z
 - - :approve
   - frontend
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2021-02-24 13:56:18.297371212 Z
 - - :approve
   - onboarding
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2021-02-24 13:56:18.596411203 Z
 - - :approve
   - polls
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2021-02-24 13:56:18.898552631 Z
 - - :approve
   - seo
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2021-02-24 13:56:19.212311410 Z
 - - :approve
   - surveys
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2021-02-24 13:56:19.512278233 Z
 - - :approve
   - volunteering
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2021-02-24 13:56:21.699541131 Z
 - - :approve
   - que-web
-  - :who: 
-    :why: 
+  - :who:
+    :why:
     :versions: []
     :when: 2021-02-24 14:15:04.392754008 Z
 - - :approve
   - project_permissions
-  - :who: 
-    :why: 
+  - :who:
+    :why:
+    :versions: []
+    :when: 2021-03-30 20:05:18.586364420 Z
+- - :approve
+  - user_confirmation
+  - :who:
+    :why:
     :versions: []
     :when: 2021-03-30 20:05:18.586364420 Z
```

Comment:
Probably not a big deal, but this value seems to be copy pasted, while in other cases it seems to have been generated each time.